examples/service-clients/azure-client/todo-list/package.json

```diff
@@ -0,0 +1,115 @@
+{
+	"name": "@fluid-example/app-integration-todo-list",
```

Comment:
Unsure about the name here. I took this from `app-intergration-external-controller`. I suspect the former was named that way because it is an example of leveraging a Fluid container to integrate into an existing application framework. This application is different. So maybe `azure-example-todo-list`?